### PR TITLE
WT-8321 Fix format-stress-smoke-test timeout condition

### PR DESCRIPTION
**Pull request by keitharnoldsmith**
Only do one format smoke test run in each Evergreen test run.

### CODE REVIEW

**Comment by keitharnoldsmith** [comment]
The one PR testing failure is WT-5917.

**Comment by keitharnoldsmith** [comment]
@lukech and @tammybailey: This PR changes the evergreen configuration so we only run the format smoke tests (i.e., `format.sh -S`) once with ASAN and once without for the relevant build variants, as suggested by Keith B on the ticket.

I've just made the easy fix -- run the tests once and shorten the evergreen timeout accordingly -- so we can eliminate these test failures.  If this seems reasonable to you two, feel free to merge it (or patch it as needed and merge) rather than waiting for me to get back online on my Monday.

This still leaves us calling the single smoke run a "stress" test and having that test configuration grouped with the other stress tests in the evergreen file.  Let me know if you have thoughts about whether/how to change that.  I'd be happy to do it in a follow-on ticket if it will get this merged sooner.

**Review by lukech — APPROVED** [review]
LGTM. I scheduled a few format smoke tasks in the PR patch build (including PPC & zSeries). 

It makes sense to move the smoke tasks out of the stress testing group after they are reduced to running only once. `!Ubuntu 20.04` and `!Ubuntu 20.04 ASAN` build variants could be their new homes. I think we can keep them out of the PR testing scope (i.e. not tagging `pull_request`) for now.

File: test/evergreen.yml [hunk 1]
```diff
@@ -647,8 +647,9 @@ variables:
 #########################################################################################
 # The following stress tests are configured to run for six hours via the "-t 360"
 # argument to format.sh: format-stress-test, format-stress-sanitizer-test, and
-# race-condition-stress-sanitizer-test. The smoke and recovery tests run in a loop,
-# with the number of runs adjusted to provide aproximately six hours of testing.
+# race-condition-stress-sanitizer-test. The recovery tests run in a loop, with
+# the number of runs adjusted to provide aproximately six hours of testing.  There
+# is just one much shorter run of the smoke tests
```
Comment (tammybailey) [inline test/evergreen.yml:652]: Missing a period here. Also, we can simply say that the smoke tests are run once (the "much shorter run" is confusing).

**Review by tammybailey — APPROVED** [review]
Looks okay to me. Just one small comment on a comment. I do think it's worth making a follow on ticket to clean up the evergreen and separate the smoke tests from the stress tests.

File: test/evergreen.yml (hunk 1 above)
Comment (keitharnoldsmith) [reply test/evergreen.yml:652]: Fixed.

Comment (keitharnoldsmith) [reply test/evergreen.yml]: I created WT-8359 for the follow-on work.

**Pull request merged**